app/main/views/services.py

```diff
@@ -151,6 +151,49 @@ def update_service(service_id):
     return jsonify(message="done"), 200
 
 
+@main.route('/services/<string:service_id>/revert', methods=['POST'])
```

Comment:
Hmmmmmmmm interesting... maybe... @TheDoubleK ? @allait ?

Though of course it's not a straight representation-transfer here.

Reply:
straight representation-transfer?

Reply:
as in ReST

Reply:
how would a straight representation-transfer look different to this?

Reply:
i.e. rather than throwing an object around here, which is where POST vs PUT is most relevant, we're just kicking a trigger saying "do something for us".

Reply:
as in, rather than handing this endpoint the data we want to PUT, we're just saying 'go find this thing with this id and do something'?

Reply:
bingo